department-of-veterans-affairs/caseflow
# "Already Certified" tweaks, OMO request changes issue modification fields

Two unrelated changes being lumped together.

### Description
- Updates Certification's "Already Certified" page content for grammatical correctness
- Updates fields on the VACOLS Decass table that we weren't previously updating

### Testing Plan
- Sign in as BVASRITCHIE 
- View tasks in Queue.
- Complete OMO request flow
- Run `VACOLS::Decass.find_by(defolder: vacols_brieff_id)` in the Rails console, verify that modified user and modified time are set
- /certifications/new/789C (in default dev/non-FACOLS mode) should look like this:
![image](https://user-images.githubusercontent.com/3781835/38153190-542f9c42-3439-11e8-90d1-cab1553b2e34.png)
